src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -84,14 +91,43 @@ private ConfigurationSetting<IList<ManifestInfo>> GetDefaultManifestInfoForValid
         }
 
         return new ConfigurationSetting<IList<ManifestInfo>>
+            {
+                Source = SettingSource.Default,
+                Value = new List<ManifestInfo>()
+                {
+                    defaultManifestInfo
+                }
+            };
+        }
+
+    private void ValidateBuildDropPathConfiguration(IConfiguration configuration)
         {
-            Source = SettingSource.Default,
-            Value = new List<ManifestInfo>()
+            if (configuration.ManifestToolAction != ManifestToolActions.Validate && configuration.ManifestDirPath?.Value != null && configuration.DockerImagesToScan?.Value != null)
```

Comment:
These conditionals are hard to read. I see they all have common condition `configuration.ManifestToolAction != ManifestToolActions.Validate`, maybe you can move it out to outer if statement, e.g.
```csharp
if (configuration.ManifestToolAction != ManifestToolActions.Validate) 
{
    if ()
    else if ()
    ...
}
```

Reply:
And it probably would be more logical to invert this top condition:
`if (configuration.ManifestToolAction == ManifestToolActions.Generate)`

Reply:
Fixed.